introduction.md

```diff
@@ -1 +1,5 @@
-My name is Daniel Stelle and I am an apprentice at Coding Zeal.  My favorite Code School course so far is the JavaScript path.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?